### ISSUE TITLE
Screen readers unable to process tables

### ISSUE THREAD

**Issue by benjaland** [adazzle/react-data-grid#1257]
### Which version of React JS are you using?
- [x] v15.4.x

### Which browser are you using?
✅ Officially supported ✅
- [x] IE 9 / IE 10 / IE 11

### I'm submitting a ...
- [x] 💡 Feature Request

### Issue Details
As the resultant data table is formed of DIV and SPAN elements, screen readers and other assistive technologies cannot use built-in "table navigation" commands to make sense of the data.

There are two solutions for this...
Replace the DIVs with the corresponding semantically correct HTML element (TABLE/TR/TH/TD) or add the correct WAI-ARIA properties.  (role=gridcell, role=grid, role=rowgroup, role=row).

Adding roles will probably be the best performace-wise however using the correct HTML elements will maximise performace.

Happy to provide examples, supporting documentation or further assistance if required.

**Comment by benjaland** [comment]
After looking at the component some more and how it only renders a small percentage of rows at any one time - I realised the solution isn't as easy as first thought!  

I'll try and put together something and submit it to the project that addresses this.
